auditbeat/module/file_integrity/metricset.go

```diff
@@ -87,7 +87,9 @@ func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
 		return nil, err
 	}
 
-	r, err := NewEventReader(config)
+	logger := logp.NewLogger(moduleName)
```

Comment:
Actually this would be a behavior change because metricbeat assumes that we have unique dataset names. So you did what I suggested we would have a logger named `file_integrity.file_integrity`. So don't do that.

But if would be nice if we could add the `id` the same as the given logger.

https://github.com/elastic/beats/blob/fe5088338011c48230e7d6aa6ce07fed5a895e5c/metricbeat/mb/builders.go#L202-L204

Reply:
ok, I did one last force push, https://github.com/elastic/beats/pull/37223/commits/04234ebd9e42698c2772d5224f9e09636a257670

let me know if this is what you meant